docs/security.rst

```diff
@@ -0,0 +1,34 @@
+Security Policy
+===============
+
+This documents the security policy and actions to take to secure the security of the package, its deployment and use.
+
+Supported Versions
+------------------
+
+Security vulnerabilities are fixed only for the latest version of `icalendar`.
+
+.. list-table:: Versions to receive security updates
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Version
+     - Supported
+   * - 6.*
+     - ✅
+   * - 5.*
+     - ❌
+   * - 4.*
+     - ❌
+   * - < 4.*
+     - ❌
+
+
+Reporting a Vulnerability
+-------------------------
+
+Please `report vulnerabilities of icalendar to Plone
+<https://github.com/plone/.github/blob/main/SECURITY.md#readme>`_.
+If you cannot do this, please contact one of the
+:ref:`maintainers`
+directly or open an issue.
```

Comment:
You should not ask people to open a security issue in a public arena, such as an issue tracker. This also conflicts with the link above, which links to https://plone.org/security/report.

If you want to create an email distribution group, and add members to it to review security, I'd suggest following Pylon Project's example as in https://groups.google.com/g/pylons-project-security/ or other free email group distribution list service that is less Google-y.

Reply:
I would opt for the GitHub process. This way, we do not need to maintain another list.